governance.md

```diff
@@ -0,0 +1,105 @@
+artifacts are all valuable ways to contribute to a Confidential Containers project.
+
+Project contributions will be reviewed by the project maintainers and should
+pass all applicable tests.
+
+## Maintainer
+
+Each project has one or more *Maintainers*. Project maintainers are first and
+foremost active *Contributors* to the project and are responsible for:
+
+* Setting technical directions for the project.
+* Facilitating, reviewing and merging contributions. They have write access to
+  the project repository.
+* Creating and assigning project issues.
+* Enforcing the [Code of Conduct](https://github.com/confidential-containers/community/blob/main/CODE_OF_CONDUCT.md).
+
+The list of maintainers for a project is defined by the project
+`CODEOWNERS` file.
+
+### Becoming a project maintainer
+
+Existing maintainers may decide to elevate a *Contributor* to the *Maintainer*
+role based on the contributor established trust and contributions relevance.
+This decision process is not formally defined and is based on lazy consensus
+from the existing maintainers.
+
+Any contributor may request for becoming a project maintainer by opening an
+pull request (PR) against the `CODEOWNERS` file, and adding *all* current maintainers
+as reviewer of the PR.
+Maintainers may also pro-actively promote contributors based on their
+contributions and leadership track record.
+
+## Steering Committee Member
+
+The Steering Committee (SC) is the overall Confidential Containers organization
+governing body.
+
+The SC provides decision-making and strategic oversight for the project. It also
+defines and enforces the project values and structure.
+
+### Scope and Responsibilities
+
+The scope and responsibilites of the SC is subject to changes, and SC members must
+adapt it to meet the project needs.
+Moreover, any technical responsibilities should be delegated to project Maintainers,
+although the SC can be consulted to help the with making technical decisions.
+
+Based on that, the SC is responsible for:
+
+* Defining the project high-level strategy and roadmap
+* Managing and administrating the project
+* Building and growing the Confidential Containers community
+* Onboarding and guiding Confidential Containers end-users
+
+### Members
+
+The initial composition of the SC is self-defined and is made of 10 members:
+
+* Larry Dewey (@larrydewey) - AMD
+* Jiang Liu (@jiangliu) and Jia Zhang (@jiazhang0) - Alibaba
+* James Magowan (@magowan)  and Tobin Feldman-Fitzthum (@fitzthum) - IBM
+* Peter Zhu (@peterzcst) and Dan Middleton (@dcmiddle) - Intel
+* Pradipta Banerjee (@bpradipt)  and Ariel Adam (@ariel-adam) - Red Hat
+* Samuel Ortiz (@sameo)
+
+#### Selection
+
+The initial SC members will be responsible for defining how and when the SC
+membership can be expanded or renewed.
+
+### Quorum
+
+Each SC member is expected to attend each SC meeting, and the following guidelines
+are used to determine if quorum is reached:
```

Comment:
I propose to have a bi-weekly (every other week) SC meeting. About reasonable times, we know very well it will always be unreasonable for at least one time zone. I think the timing  should be left for the SC members to decide and agree upon and as such it may change over time.
Also, yes those meetings should be public and recorded, I think.

Reply:
Holding a bi-weekly meeting makes sense to me, as well. I would be open to possibly discussing an alternating meeting schedule, as well. I don't want to over complicate or make the meeting times too confusing, but it might be nice for those for which the meeting is less convenient?

Reply:
I am open to alternating schedule - though I appreciate that being in Europe I am perhaps least affected by possible meeting times.